CHANGELOG.md

```diff
@@ -1,3 +1,14 @@
+### v0.33.1
+###### Features
+- Add Python 3.9 testing (pull #[1584](https://github.com/aws-cloudformation/cfn-python-lint/pull/1584))
+###### CloudFormation Specifications
+- Update CloudFormation specs to 15.0.0 (pull #[1582](https://github.com/aws-cloudformation/cfn-python-lint/pull/1582))
+- Add additional allowed values for `AWS::SecretsManager::SecretTargetAttachment.TargetType` (pull #[1573](https://github.com/aws-cloudformation/cfn-python-lint/pull/1573))
+- Add property types for `AvailabilityZone` in the `AWS::DMS::ReplicationInstance` and `AWS::EC2::Subnet` resources (pull #[1585](https://github.com/aws-cloudformation/cfn-python-lint/pull/1585))
```

Comment:
```suggestion
```

not sure this made it into [`CloudSpecs`](https://github.com/aws-cloudformation/cfn-python-lint/tree/master/src/cfnlint/data/CloudSpecs) to actually be in effect yet

Reply:
Updated specs so its included now.